diff --git a/node_modules/react-native-status-bar-height/index.js b/node_modules/react-native-status-bar-height/index.js
index 08fd56b..74c2ee2 100644
--- a/node_modules/react-native-status-bar-height/index.js
+++ b/node_modules/react-native-status-bar-height/index.js
@@ -16,6 +16,10 @@ const IP12_HEIGHT = 844;
 
 const IP12MAX_WIDTH = 428;
 const IP12MAX_HEIGHT = 926;
+const IP14MAX_WIDTH = 430;
+const IP14MAX_HEIGHT = 932;
+const IP14_WIDTH = 393;
+const IP14_HEIGHT = 852;
 
 const { height: W_HEIGHT, width: W_WIDTH } = Dimensions.get('window');
 
@@ -27,7 +31,15 @@ let isIPhone12Max_v = false;
 let isIPhoneWithMonobrow_v = false;
 
 if (Platform.OS === 'ios' && !Platform.isPad && !Platform.isTVOS) {
-    if (W_WIDTH === X_WIDTH && W_HEIGHT === X_HEIGHT) {
+    if (W_WIDTH === IP14_WIDTH && W_HEIGHT === IP14_HEIGHT) {
+        isIPhoneWithMonobrow_v = true;
+        isIPhoneX_v = true;
+        statusBarHeight = STATUSBAR_X_HEIGHT;
+    } else if (W_WIDTH === IP14MAX_WIDTH && W_HEIGHT === IP14MAX_HEIGHT) {
+        isIPhoneWithMonobrow_v = true;
+        isIPhoneX_v = true;
+        statusBarHeight = STATUSBAR_X_HEIGHT;
+    } else if (W_WIDTH === X_WIDTH && W_HEIGHT === X_HEIGHT) {
         isIPhoneWithMonobrow_v = true;
         isIPhoneX_v = true;
         statusBarHeight = STATUSBAR_X_HEIGHT;
@@ -46,10 +58,10 @@ if (Platform.OS === 'ios' && !Platform.isPad && !Platform.isTVOS) {
     }
 }
 
-export const isIPhoneX = () =>  isIPhoneX_v;
-export const isIPhoneXMax = () =>  isIPhoneXMax_v;
-export const isIPhone12 = () =>  isIPhone12_v;
-export const isIPhone12Max = () =>  isIPhone12Max_v;
+export const isIPhoneX = () => isIPhoneX_v;
+export const isIPhoneXMax = () => isIPhoneXMax_v;
+export const isIPhone12 = () => isIPhone12_v;
+export const isIPhone12Max = () => isIPhone12Max_v;
 export const isIPhoneWithMonobrow = () => isIPhoneWithMonobrow_v;
 
 const getExpoRoot = () => global.Expo || global.__expo || global.__exponent;
